add/script.rpy

```diff
@@ -1606,13 +1606,13 @@ translate french afterTalkWithKira_ba6733e0:
 translate french afterTalkWithKira_bffdf4d4:
 
     # me "You aren't going to tell me, are you?"
-    me "Tu ne vas pas me le dire, n'est-ce pas ?"
+    me "Mais tu me le diras pas."

```

Comment:
J'ai enlevé le "ne" que proposait Otto, mais le reste de la phrase semblait un peu étrange. C'est fréquent en anglais de poser de fausse question qui finisse par are/aren't you. 
Je me suis dis qu'en francais, je repondrais surement ca, par une affirmation dans ce style